src/App.tsx

```diff
@@ -12,34 +12,72 @@ function App() {
 }
 
 const Body = () => {
+  const [chart, setChart] = useState("line");
```

Comment:
Can we rename this to `chartType` and `setChartType` since this is what we are storing here?

Reply:
Done.